L103-core-move-insecure-creds-declaration.md

```diff
@@ -0,0 +1,36 @@
+Move function declarations for insecure credentials from `grpc/grpc_security.h`
+----
+* Author(s): [Cheng-Yu Chung (@ralphchung)](https://github.com/ralphchung)
+* Approver: [@ctiller](https://github.com/ctiller), [@markdroth](https://github.com/markdroth), [@mehrdada](https://github.com/mehrdada), [@drfloob](https://github.com/drfloob)
+* Status: Ready for Implementation
+* Implemented in: C Core
+* Last updated: 09/28/2022
+* Discussion at: https://groups.google.com/g/grpc-io/c/6qvo-UVs-uI
+
+## Abstract
+
+Move function declarations for insecure credentials from `grpc/grpc_security.h` to a new header file `grpc/grpc_insecure_credentials.h`.
+
+## Background
+
+According to the discussion in https://github.com/grpc/grpc/issues/31012, it makes no sense to include `grpc/grpc_security.h` if the file has nothing to do with secure credentials.
```

Comment:
I think it's also worth mentioning that this builds on work done in grpc/grpc#25586 and helps move us to an eventual future where each type of credentials is in its own build target, so applications can link in only the specific one(s) they need.